### ISSUE TITLE
Support Windows again

### ISSUE THREAD

**Issue by raehik** [camfort/camfort#139]
Windows support waned around v0.905 with the introduction of Flint. CamFort is a user-facing CLI tool, so it'd be nice to re-introduce support if possible to keep CamFort's audience wide.

[camfort/fortran-src](https://github.com/camfort/fortran-src) already builds and tests on Windows, so no problems there. The primary topics are:

  * external C library usage, building, linking
  * file & filepath-related tests having different behaviour
  * distribution (z3? Flint+GMP+BLAS?)

**Comment by raehik** [comment]
There was interest in building for Windows back in Nov 2020 @ #119 and it's come up a few times during my post. I've made progress recently (finally!), so will keep moving.

**Comment by raehik** [comment]
Fixes for runtime crashes during testing almost prepared @ #138

**Comment by DrK** [comment]
I downloaded yesterday and it all seems to work, except --version reports 1.0, and I was expecting 1.1

**Comment by raehik** [comment]
@DrK Yes, I haven't been updating the internal version string. I was looking for an excuse to test the releasing once more, so I'll update that.
